client/src/components/Messages.jsx

```diff
@@ -12,10 +12,22 @@ import Message from './Message.jsx';
 import UserContacted from './UserContacted.jsx';
 import dummyData from '../../../messagesDummyData';
 
+import { AuthContext } from '../contexts/index.jsx';
+
+
 function Messages({ userId }) {
   const [usersContacted, setUsersContacted] = useState(dummyData.usersContacted);
   const [messages, setMessages] = useState(dummyData.messages);
   const [userContactedId, setUserContactedId] = useState(null);
+  //  alex test
```

Comment:
Any chance we can clean these comments up before merging?

Reply:
removing the comments now, I meant to removre